### ISSUE TITLE
Kali binary package: improvements

### ISSUE THREAD

**Issue by revast** [v1s1t0r1sh3r3/airgeddon#317]
hello,
airgeddon_9.23-1_all/usr/local/bin/airgeddon should be in:
airgeddon_9.23-1_all/usr/bin/airgeddon

DEBIAN/changelog and DEBIAN/changelog should be in:
 airgeddon_9.23-1_all/usr/share/doc/airgeddon/changelog 
 airgeddon_9.23-1_all/usr/share/doc/airgeddon/copyright

the airgeddon_9.23-1_all/usr/share/applications/kali-airgeddon.desktop does not need to be maked as executable.

**Comment by v1s1t0r1sh3r3** [comment]
Calling these bugs is brave.... improvements? suggestions? could you edit please?

**Comment by OscarAkaElvis** [comment]
Checking this deeply... 

> airgeddon_9.23-1_all/usr/local/bin/airgeddon should be in:
airgeddon_9.23-1_all/usr/bin/airgeddon

Why? both paths are in PATH and are able to be executed from anywhere. I think that if someday we at last are added to Kali repos, they will do however more correct is for them, and this binary will disappear from airgeddon repo. Anyway, I think I can understand your point. This can be changed, it will have not too much effect.

> /usr/share/applications/kali-airgeddon.desktop does not need to be maked as executable.

So purist man! 🐙 . Maybe you are right, easy to change to 644 permissions. I also changed on icon and on other files like LICENSE file, etc.

> airgeddon_9.23-1_all/usr/share/doc/airgeddon/changelog
airgeddon_9.23-1_all/usr/share/doc/airgeddon/copyright

Ok that can be done easily for next version. I already did some tests and everything went fine.

Ok, you talked about debian docs... I guess you think airgeddon docs  are correctly at `/usr/share/airgeddon/docs`. I'm talking about README.md , CHANGELOG.md, CODE_OF_CONDUCT.md , etc. LICENSE file is ok at `/usr/share/airgeddon/licenses`.

All the changes will be on the next v10.0

P.S. I edited your title for better understanding. Thanks for your suggestion. Closing.
